index.js

```diff
@@ -78,6 +82,53 @@ class TrezorKeyring extends EventEmitter {
     return this.model;
   }
 
+  /**
+   * Gets the vendor, if known.
+   *
+   * @returns {"trezor" | "onekey"}
+   */
+  getVendor() {
+    return this.vendor;
+  }
+
+  /**
+   * set the vendor name of the hardware wallet
+   * @param {*} features
+   * @returns {'onekey' | 'trezor'}
+   */
+  __setVendor(features) {
+    // If the value of features is null, set vendor to the default value
+    if (features === null) {
```

Comment:
Why is there different behavior when `features === null` vs `!features`? Could they be taken care of as one case?
 ```suggestion
    if (!features) {
```